www/class/centreonAuth.SSO.class.php

```diff
@@ -103,7 +102,18 @@ public function __construct(
             # Get configured values
             $clientId = $this->ssoOptions['openid_connect_client_id'];
             $clientSecret = $this->ssoOptions['openid_connect_client_secret'];
-            $redirectNoEncode = $this->ssoOptions['openid_connect_redirect_url'];
+            if (empty($this->ssoOptions['openid_connect_redirect_url'])) {
+                $redirectNoEncode = '{$scheme}://{$hostname}:{$port}'
+                . $this->ssoOptions['oreon_web_path'] . 'index.php';
```

Comment:
use rtrim to remove potential '/' defined by oreon_web_path and add a "/" in concatenation

Reply:
fix with PR #10098 10098